remill/Arch/X86/Semantics/BITBYTE.cpp

```diff
@@ -253,6 +253,11 @@ DEF_ISEL_MnW_Mn_Rn(BTC_MEMv_GPRv, BTCmem);
 DEF_ISEL_RnW_Rn_Rn(BTC_GPRv_GPRv, BTCreg);
 
 namespace {
+DEF_SEM(BSWAP_16, R16W dst, R16 src) {
+  Write(dst, (int16_t)0);
```

Comment:
Don't use C-style casts. Perhaps instead even use `__remill_undefined_16()` in place of zero.